README.md

```diff
@@ -256,9 +262,9 @@ The following roadmap covers the main features and ideas we want to implement bu
 
 - [x] Support Erigon on Gnosis
 - [x] Support for Lido CSM
+- [x] Support for Nimbus client as Consensus and Validator
 - [ ] Include monitoring tool for alerting, tracking validator balance, and tracking sync progress and status of nodes
```

Comment:
We are delivering this in this release :)

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/456/commits/466d65f5b91ba8357e450b3025905668920e7285)